control_panel_api/views.py

```diff
@@ -170,7 +166,8 @@ class AppCustomersDetailAPIView(GenericAPIView):
     permission_classes = (IsSuperuser,)
 
     def delete(self, request, *args, **kwargs):
-        Auth0().delete_group_member(self.get_object().slug, kwargs['user_id'])
+        app = self.get_object()
+        app.delete_customers([kwargs['user_id']])
```

Comment:
So `delete` is still done one by one 👍

Reply:
Yep, I just changed the API to make it possible to do bulk adds and deletions, but I haven't changed the views.